components/resources/brave_shields_strings.grdp

```diff
@@ -66,43 +66,43 @@
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_TRACKERS_AND_ADS_BLOCKED_STD" desc="Menu option that would block some trackers and advertisement website scripts.">
-    Trackers &amp; ads blocked (standard)
+    Block trackers &amp; ads
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_TRACKERS_AND_ADS_BLOCKED_AGG" desc="Menu option that would block all trackers and advertisement website scripts.">
-    Trackers &amp; ads blocked (aggressive)
+    Aggressively block trackers &amp; ads
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_TRACKERS_AND_ADS_ALLOW_ALL" desc="Menu option that would stop blocking all trackers and advertisement website scripts">
     Allow all trackers &amp; ads
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_CROSS_COOKIES_BLOCKED" desc="Menu option that would block cross website cookies.">
-    Cross-site cookies blocked
+    Block cross-site cookies
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_COOKIES_BLOCKED" desc="Menu option that would block current website cookies.">
-    Cookies blocked
+    Block all cookies
   </message>
 
-  <message name="IDS_BRAVE_SHIELDS_COOKIES_BLOCKED_ALL" desc="Menu option that would allow all cookies.">
-    All cookies allowed
+  <message name="IDS_BRAVE_SHIELDS_COOKIES_BLOCKED_ALL" desc="Menu option that would allow all cookies on the current website.">
```

Comment:
```suggestion
  <message name="IDS_BRAVE_SHIELDS_COOKIES_ALLOWED_ALL" desc="Menu option that would allow all cookies on the current website.">
```

I don't know if you can safely change the ID so easily without breaking other things, but consider changing it as it is confusing right now.